app/components/Solar-Drought-Visualizer/MapboxMap.tsx

```diff
@@ -0,0 +1,99 @@
+'use client'
+
+import 'mapbox-gl/dist/mapbox-gl.css'
+import 'react-map-gl-geocoder/dist/mapbox-gl-geocoder.css'
+import React, { useRef, useState, useEffect } from 'react'
+import { Marker, Map, Layer, Source, MapMouseEvent, NavigationControl } from 'react-map-gl'
+import { Button } from '@mui/material'
```

Comment:
@cohenlea I try to keep this section organized because it can get crazy

-React and library imports
-Material UI imports
-Local imports

I'll merge for now but just so we can keep in mind